Content.Client/UserInterface/HumanoidProfileEditor.cs

```diff
@@ -360,7 +361,33 @@ public HumanoidProfileEditor(IClientPreferencesManager preferencesManager, IProt
                     appearanceVBox.AddChild(panel);
                 }
 
-                #endregion Clothing
+                #endregion Backpack
+
+                #region Eyes
+
+                {
+                    var panel = HighlightedContainer();
+                    panel.SizeFlagsHorizontal = SizeFlags.None;
```

Comment:
`panel.SizeFlagsHorizontal = SizeFlags.None;`

should be 

`panel.SizeFlagsHorizontal = SizeFlags.FillExpand;`

Like PJB said.